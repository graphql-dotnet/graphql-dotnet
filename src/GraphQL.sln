
Microsoft Visual Studio Solution File, Format Version 12.00
# Visual Studio Version 16
VisualStudioVersion = 16.0.29424.173
MinimumVisualStudioVersion = 10.0.40219.1
Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{C1C3FFFD-3369-42B1-BE2E-CA10AD9E2ADA}"
	ProjectSection(SolutionItems) = preProject
		..\.editorconfig = ..\.editorconfig
		..\appveyor.yml = ..\appveyor.yml
		CommonAssemblyInfo.cs = CommonAssemblyInfo.cs
		Directory.Build.props = Directory.Build.props
		NuGet.config = NuGet.config
		..\README.md = ..\README.md
		Tests.props = Tests.props
	EndProjectSection
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL", "GraphQL\GraphQL.csproj", "{1DD5E10B-2361-467A-82AC-36EFB8C61A1D}"
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.Tests", "GraphQL.Tests\GraphQL.Tests.csproj", "{65E750AC-4701-45A8-97C0-822B1CC42C1E}"
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.DataLoader.Tests", "GraphQL.DataLoader.Tests\GraphQL.DataLoader.Tests.csproj", "{21FA4059-CF8D-4A5B-8C1B-D4DB645311B9}"
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.Harness", "GraphQL.Harness\GraphQL.Harness.csproj", "{00D40874-DCB7-4D44-B2B0-F88EB1FAA28E}"
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.StarWars", "GraphQL.StarWars\GraphQL.StarWars.csproj", "{11A414C3-7847-4852-8A62-DCF8A438B938}"
EndProject
Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{BC80C071-B8B6-42D2-9CF8-6169C6169B74}"
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.Harness.Tests", "GraphQL.Harness.Tests\GraphQL.Harness.Tests.csproj", "{CB5C204A-FBE9-40CC-B4C7-A29E7E162DBE}"
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.Dummy", "GraphQL.Dummy\GraphQL.Dummy.csproj", "{17CA647F-903F-4A0A-BF8F-E5123AAA2617}"
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.ApiTests", "GraphQL.ApiTests\GraphQL.ApiTests.csproj", "{1CEB8BF8-2292-4B93-9C58-860887FEC9D4}"
EndProject
Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Docs", "Docs", "{EB92DFCF-A87E-4E8B-B068-8B396D00945A}"
EndProject
Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "getting-started", "getting-started", "{587B3CCD-8B68-491B-9AC5-0E04DF3E9507}"
	ProjectSection(SolutionItems) = preProject
		..\docs2\site\docs\getting-started\aliases.md = ..\docs2\site\docs\getting-started\aliases.md
		..\docs2\site\docs\getting-started\arguments.md = ..\docs2\site\docs\getting-started\arguments.md
		..\docs2\site\docs\getting-started\authorization.md = ..\docs2\site\docs\getting-started\authorization.md
		..\docs2\site\docs\getting-started\custom-scalars.md = ..\docs2\site\docs\getting-started\custom-scalars.md
		..\docs2\site\docs\getting-started\databases.md = ..\docs2\site\docs\getting-started\databases.md
		..\docs2\site\docs\getting-started\dependency-injection.md = ..\docs2\site\docs\getting-started\dependency-injection.md
		..\docs2\site\docs\getting-started\directives.md = ..\docs2\site\docs\getting-started\directives.md
		..\docs2\site\docs\getting-started\errors.md = ..\docs2\site\docs\getting-started\errors.md
		..\docs2\site\docs\getting-started\field-middleware.md = ..\docs2\site\docs\getting-started\field-middleware.md
		..\docs2\site\docs\getting-started\fields.md = ..\docs2\site\docs\getting-started\fields.md
		..\docs2\site\docs\getting-started\fragments.md = ..\docs2\site\docs\getting-started\fragments.md
		..\docs2\site\docs\getting-started\graphiql.md = ..\docs2\site\docs\getting-started\graphiql.md
		..\docs2\site\docs\getting-started\installation.md = ..\docs2\site\docs\getting-started\installation.md
		..\docs2\site\docs\getting-started\interfaces.md = ..\docs2\site\docs\getting-started\interfaces.md
		..\docs2\site\docs\getting-started\introduction.md = ..\docs2\site\docs\getting-started\introduction.md
		..\docs2\site\docs\getting-started\lists-non-null.md = ..\docs2\site\docs\getting-started\lists-non-null.md
		..\docs2\site\docs\getting-started\malicious-queries.md = ..\docs2\site\docs\getting-started\malicious-queries.md
		..\docs2\site\docs\getting-started\metadata.md = ..\docs2\site\docs\getting-started\metadata.md
		..\docs2\site\docs\getting-started\metrics.md = ..\docs2\site\docs\getting-started\metrics.md
		..\docs2\site\docs\getting-started\mutations.md = ..\docs2\site\docs\getting-started\mutations.md
		..\docs2\site\docs\getting-started\queries.md = ..\docs2\site\docs\getting-started\queries.md
		..\docs2\site\docs\getting-started\query-organization.md = ..\docs2\site\docs\getting-started\query-organization.md
		..\docs2\site\docs\getting-started\query-validation.md = ..\docs2\site\docs\getting-started\query-validation.md
		..\docs2\site\docs\getting-started\relay.md = ..\docs2\site\docs\getting-started\relay.md
		..\docs2\site\docs\getting-started\schema-types.md = ..\docs2\site\docs\getting-started\schema-types.md
		..\docs2\site\docs\getting-started\subscriptions.md = ..\docs2\site\docs\getting-started\subscriptions.md
		..\docs2\site\docs\getting-started\unions.md = ..\docs2\site\docs\getting-started\unions.md
		..\docs2\site\docs\getting-started\user-context.md = ..\docs2\site\docs\getting-started\user-context.md
		..\docs2\site\docs\getting-started\variables.md = ..\docs2\site\docs\getting-started\variables.md
	EndProjectSection
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.Benchmarks", "GraphQL.Benchmarks\GraphQL.Benchmarks.csproj", "{F08D07D5-1BD7-41A7-8F1B-FF655517A058}"
EndProject
Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "guides", "guides", "{F279F263-BCEB-4738-9111-C85139BC63A7}"
	ProjectSection(SolutionItems) = preProject
		..\docs2\site\docs\guides\dataloader.md = ..\docs2\site\docs\guides\dataloader.md
		..\docs2\site\docs\guides\links.md = ..\docs2\site\docs\guides\links.md
		..\docs2\site\docs\guides\migration.md = ..\docs2\site\docs\guides\migration.md
		..\docs2\site\docs\guides\schema-generation.md = ..\docs2\site\docs\guides\schema-generation.md
	EndProjectSection
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.NewtonsoftJson", "GraphQL.NewtonsoftJson\GraphQL.NewtonsoftJson.csproj", "{55968205-5F9A-4646-AFE4-F29913570DBA}"
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.SystemTextJson", "GraphQL.SystemTextJson\GraphQL.SystemTextJson.csproj", "{FCE8E24C-29AA-4AED-93BA-08551D38823A}"
EndProject
Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Serialization", "Serialization", "{7BA2B19D-2839-4C7D-9BB0-4F3ADE691ADD}"
EndProject
Global
	GlobalSection(SolutionConfigurationPlatforms) = preSolution
		Debug|Any CPU = Debug|Any CPU
		Release|Any CPU = Release|Any CPU
	EndGlobalSection
	GlobalSection(ProjectConfigurationPlatforms) = postSolution
		{1DD5E10B-2361-467A-82AC-36EFB8C61A1D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{1DD5E10B-2361-467A-82AC-36EFB8C61A1D}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{1DD5E10B-2361-467A-82AC-36EFB8C61A1D}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{1DD5E10B-2361-467A-82AC-36EFB8C61A1D}.Release|Any CPU.Build.0 = Release|Any CPU
		{65E750AC-4701-45A8-97C0-822B1CC42C1E}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{65E750AC-4701-45A8-97C0-822B1CC42C1E}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{65E750AC-4701-45A8-97C0-822B1CC42C1E}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{65E750AC-4701-45A8-97C0-822B1CC42C1E}.Release|Any CPU.Build.0 = Release|Any CPU
		{21FA4059-CF8D-4A5B-8C1B-D4DB645311B9}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{21FA4059-CF8D-4A5B-8C1B-D4DB645311B9}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{21FA4059-CF8D-4A5B-8C1B-D4DB645311B9}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{21FA4059-CF8D-4A5B-8C1B-D4DB645311B9}.Release|Any CPU.Build.0 = Release|Any CPU
		{00D40874-DCB7-4D44-B2B0-F88EB1FAA28E}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{00D40874-DCB7-4D44-B2B0-F88EB1FAA28E}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{00D40874-DCB7-4D44-B2B0-F88EB1FAA28E}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{00D40874-DCB7-4D44-B2B0-F88EB1FAA28E}.Release|Any CPU.Build.0 = Release|Any CPU
		{11A414C3-7847-4852-8A62-DCF8A438B938}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{11A414C3-7847-4852-8A62-DCF8A438B938}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{11A414C3-7847-4852-8A62-DCF8A438B938}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{11A414C3-7847-4852-8A62-DCF8A438B938}.Release|Any CPU.Build.0 = Release|Any CPU
		{CB5C204A-FBE9-40CC-B4C7-A29E7E162DBE}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{CB5C204A-FBE9-40CC-B4C7-A29E7E162DBE}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{CB5C204A-FBE9-40CC-B4C7-A29E7E162DBE}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{CB5C204A-FBE9-40CC-B4C7-A29E7E162DBE}.Release|Any CPU.Build.0 = Release|Any CPU
		{17CA647F-903F-4A0A-BF8F-E5123AAA2617}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{17CA647F-903F-4A0A-BF8F-E5123AAA2617}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{17CA647F-903F-4A0A-BF8F-E5123AAA2617}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{17CA647F-903F-4A0A-BF8F-E5123AAA2617}.Release|Any CPU.Build.0 = Release|Any CPU
		{1CEB8BF8-2292-4B93-9C58-860887FEC9D4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{1CEB8BF8-2292-4B93-9C58-860887FEC9D4}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{1CEB8BF8-2292-4B93-9C58-860887FEC9D4}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{1CEB8BF8-2292-4B93-9C58-860887FEC9D4}.Release|Any CPU.Build.0 = Release|Any CPU
		{F08D07D5-1BD7-41A7-8F1B-FF655517A058}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{F08D07D5-1BD7-41A7-8F1B-FF655517A058}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{F08D07D5-1BD7-41A7-8F1B-FF655517A058}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{F08D07D5-1BD7-41A7-8F1B-FF655517A058}.Release|Any CPU.Build.0 = Release|Any CPU
		{55968205-5F9A-4646-AFE4-F29913570DBA}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{55968205-5F9A-4646-AFE4-F29913570DBA}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{55968205-5F9A-4646-AFE4-F29913570DBA}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{55968205-5F9A-4646-AFE4-F29913570DBA}.Release|Any CPU.Build.0 = Release|Any CPU
		{FCE8E24C-29AA-4AED-93BA-08551D38823A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{FCE8E24C-29AA-4AED-93BA-08551D38823A}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{FCE8E24C-29AA-4AED-93BA-08551D38823A}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{FCE8E24C-29AA-4AED-93BA-08551D38823A}.Release|Any CPU.Build.0 = Release|Any CPU
	EndGlobalSection
	GlobalSection(SolutionProperties) = preSolution
		HideSolutionNode = FALSE
	EndGlobalSection
	GlobalSection(NestedProjects) = preSolution
		{65E750AC-4701-45A8-97C0-822B1CC42C1E} = {BC80C071-B8B6-42D2-9CF8-6169C6169B74}
		{21FA4059-CF8D-4A5B-8C1B-D4DB645311B9} = {BC80C071-B8B6-42D2-9CF8-6169C6169B74}
		{CB5C204A-FBE9-40CC-B4C7-A29E7E162DBE} = {BC80C071-B8B6-42D2-9CF8-6169C6169B74}
		{17CA647F-903F-4A0A-BF8F-E5123AAA2617} = {BC80C071-B8B6-42D2-9CF8-6169C6169B74}
		{1CEB8BF8-2292-4B93-9C58-860887FEC9D4} = {BC80C071-B8B6-42D2-9CF8-6169C6169B74}
		{587B3CCD-8B68-491B-9AC5-0E04DF3E9507} = {EB92DFCF-A87E-4E8B-B068-8B396D00945A}
		{F279F263-BCEB-4738-9111-C85139BC63A7} = {EB92DFCF-A87E-4E8B-B068-8B396D00945A}
		{55968205-5F9A-4646-AFE4-F29913570DBA} = {7BA2B19D-2839-4C7D-9BB0-4F3ADE691ADD}
		{FCE8E24C-29AA-4AED-93BA-08551D38823A} = {7BA2B19D-2839-4C7D-9BB0-4F3ADE691ADD}
	EndGlobalSection
	GlobalSection(ExtensibilityGlobals) = postSolution
		SolutionGuid = {61AAA7F9-9D46-4050-A3A5-F4358BBD95B8}
	EndGlobalSection
EndGlobal
