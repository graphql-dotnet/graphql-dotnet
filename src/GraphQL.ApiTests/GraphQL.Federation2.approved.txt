namespace GraphQL.Federation.Instrumentation
{
    public static class FederatedTracingExtensions
    {
        public static void EnrichWithApolloFederatedTracing(this GraphQL.ExecutionResult executionResult, System.DateTime start) { }
    }
}
namespace GraphQL
{
    public static class FederationGraphQLBuilderExtensions
    {
        public static GraphQL.DI.IGraphQLBuilder UseApolloFederatedTracing(this GraphQL.DI.IGraphQLBuilder builder, bool enableMetrics = true) { }
        public static GraphQL.DI.IGraphQLBuilder UseApolloFederatedTracing(this GraphQL.DI.IGraphQLBuilder builder, System.Func<GraphQL.ExecutionOptions, bool> enableMetricsPredicate) { }
    }
}
